pkgs/applications/science/robotics/mavproxy/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, matplotlib, numpy, pymavlink, pyserial
+, setuptools, wxPython_4_0 }:
+
+buildPythonApplication rec {
+  pname = "MAVProxy";
+  version = "1.8.17";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "193hjilsmbljbgj7v6icy3b4hzm14l0z6v05v7ycx6larij5xj2r";
+  };
+
+  propagatedBuildInputs = [ matplotlib numpy pymavlink pyserial setuptools
```

please do a package per-line if it's this long
```suggestion
  propagatedBuildInputs = [ 
    matplotlib 
    numpy 
    pymavlink
     pyserial 
     setuptools
```